### ISSUE TITLE
Header/footer toolbar slot support ?

### ISSUE THREAD

**Issue by lllopo** [fullcalendar/fullcalendar-vue#116]
Is there header/footer toolbar slot support ? If not - it would be very, very much welcome. 
I'm building a Vue app around FullCalendar, that is really responsive - it switches the FC views in a way that only those that fit the current screen size are available and this works great.  The rpoblem is that the header layout itself is not properly responsive. For now I have a custom solution outside FC that interacts with FC events to be in sync, but this adds a lot of coding overhead and is frankly cumbersome. In addition - I'm using bootstrap, but NOT FontAwesome, as I just inject the icons SVG's where necessary. This works fine everywhere, except FC's header, where I have to rely on FontAwesome classes to render button icons. In short - adding headers slot would allow even greater flexibility to FC and honestly - If it is not there, I don't understand how it was missed.

**Comment by acerix** [comment]
Vue slots can be used for any of the `*Content` options in the API, so are you looking for something like `headerContent` and `footerContent` to set the HTML in those places?  

For now, you should be able to the icons in `viewDidMount` .

**Comment by acerix** [comment]
Yes, seems it could be useful feature but not specific to Vue so I'll move to the main repo.
